ci/docker-compose.yml

```diff
@@ -15,7 +15,7 @@ services:
     depends_on:
       - db
   db:
-    image: postgres:11
+    image: postgres:15
```

Comment:
The ticket is for Dev, not all environments.

Reply:
This is local only.  AWS doesn't use it.